extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/VanillaKubernetesProcessor.java

```diff
@@ -293,7 +296,7 @@ public List<DecoratorBuildItem> createDecorators(ApplicationInfoBuildItem applic
 
         // Handle init Containers and Jobs
         result.addAll(KubernetesCommonHelper.createInitContainerDecorators(KUBERNETES, name, initContainers, result));
-        result.addAll(KubernetesCommonHelper.createInitJobDecorators(KUBERNETES, name, jobs, result));
```

Comment:
Why?

Reply:
Good catch, not sure how that happened but it definitely didn't happen on purpose.

Reply:
Restored.